docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -857,6 +857,83 @@
       </sect3>
     </sect2>
 
+    <sect2 id="Egress-HYAML" role="brk">
+      <title>Egress</title>
+
+      <para>In order to allow clusters to be securely locked down but still
+      allow access to the services they need there is an egress mechanism.
+      Egress provides a similar mechanism to Ingress, by being able to define
+      which endpoints and ports components are permitted to connect to.</para>
+
+      <para>Most HPCC Systems components have their own auto-generated network
+      policies. The generated network policies typically work to limit ingress
+      and egress to inter-component communication, or expected external
+      service ingress only.</para>
+
+      <para>For instance, in a default deployed system with network policies
+      enforced, a query running (on hThor, Thor, or Roxie), will not be able
+      to connect with a 3rd party service, such as an LDAP service or log
+      stack. This allows any pod with Kube API access, which doesn't have its
```

Comment:
> This allows...

Doesn't sound quite right. The previous sentence is about intentional lockdown, it's not that 'this' allows any pod with ...

I'd also combine some of the other text (e.g. from 930-934) and remove the generated NetworkPolicy below (885-895), because given the other statements, showing the internal unconfigured NetworkPolicy doesn't really clarify things.

This is my suggested revised text:

"In the default configuration, any pod with Kube API access will also have access to any external endpoint. This is because a NetworkPolicy is generated for egress access for components that need access to the Kube API. However, global.egress.kubeApiCidr and global.egress.kubeApiPort in values.yaml should be configured in a secure system to lock this egress access down so that it only exposes egress access to the Kube API endpoint."

Then continuing with your existing text:

   <para>For more information, please see the <emphasis>egress:</emphasis>
      section in the default stock/delivered HPCC Systems YAML file. The
      <emphasis>values.yaml</emphasis> file can be found under the helm/hpcc/
      directory on the HPCC Systems github repository:</para>

Then:

Here is an example of adding a custom egress definition:
```
global:
  egress:
    engineEgress:
    - to:
        - ipBlock:
            cidr: 201.13.21.0/24
        - ipBlock:
            cidr: 142.250.187.0/24
      ports:
        - protocol: TCP
          port: 6789
        - protocol: TCP
          port: 7890
```
NB: the name 'engineEgress' is an arbitrary name, any name can be chosen, and any number of these named egress sections can be defined.

Reply:
I think I followed this change accurately.